src/index.js

```diff
@@ -1 +1,43 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+const scoreDiv = document.querySelector('.score-display');
+const addGame = document.querySelector('#gameSubmit');
+const Newname = document.getElementById('name');
+const Newscore = document.getElementById('score');
+const refresh = document.getElementById('refresh');
+let output = '';
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/MgJ4HogVRPPfSuFrrM0n/scores/';
+
+// get game scores
+const renderScore = (gameUsers) => {
+  const gamers = (gameUsers.result);
+  gamers.forEach((gameUser) => {
+    output += `
+     <span>${gameUser.user}:${gameUser.score}</span>  `;
+  });
+  scoreDiv.innerHTML = output;
+};
+refresh.addEventListener('click', () => {
+  fetch(url)
+    .then((res) => res.json())
+    .then((data) => renderScore(data));
+});
```

Comment:
- [ ] **Required:**You have done an amazing job. I have observed, however, that the list behaves in an abnormal way. When a score item is added and the `refresh` button clicked, the list duplicates. Kindly fix this behavior.

https://user-images.githubusercontent.com/27709832/169272934-7db931a2-32c5-4abb-906e-bd4d66e3ec6b.mp4